spec/book_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../book'
+
+RSpec.describe Book do
+  let(:book) { Book.new('The Review and Herald', 'Best') }
+
+  describe '#new' do
+    it 'Should create a new book object' do
+      expect(book).to be_an_instance_of Book
+    end
+  end
+  describe '#Item_kind' do
+    it 'Should be a kind of Item' do
+      expect(book).to be_kind_of Item
+    end
+  end
+  describe '#publisher' do
+    it 'Should return the publisher to be The Review and Herald' do
+      expect(book.publisher).to eq 'The Review and Herald'
+    end
+  end
+  describe '#cover_state' do
+    it 'Should return the cover state to be best' do
+      expect(book.cover_state).to eq 'Best'
+    end
+  end
+end
```

Comment:
- [ ] Good job writting your tests. However, the test for can_be_archived in the book_spec is not written. Kindly note that you are required to Add unit tests for all implemented methods as per the project requirements. KIndly ensure that you implement this.

![image](https://user-images.githubusercontent.com/86318284/229469835-f9c26566-b057-48c4-87d8-4b3fe9dc8fb1.png)